nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -152,6 +152,19 @@
           compatibilty, but will be removed at a later date.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          If you are using Wayland you can choose to use the Ozone
+          Wayland support in Chrome and several Electron apps by setting
+          the environment variable <literal>NIXOS_OZONE=1</literal> (for
```

Comment:
Not sure if `NIXOS_OZONE` is the best name because this is about Ozone/Wayland. Ozone is a platform abstraction layer and e.g. Ozone/X11 is the current default (the non-Ozone X11 implementation was removed). See https://chromium.googlesource.com/chromium/src/+/HEAD/docs/ozone_overview.md for more information.

Setting `NIXOS_OZONE` (or `NIXOS_OZONE_PLATFORM`) to the Ozone platform instead of `1` might be more helpful (-> `--add-flags "\''${NIXOS_OZONE:+\''--enable-features=UseOzonePlatform --ozone-platform=$NIXOS_OZONE}"`).

And just FYI: There's also a `--ozone-platform-hint` hint flag now that supports `auto` (but it should take a while until that's supported by all Electron apps so we probably shouldn't use it yet).